test/utility.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+  SLB.Utility
+
+  Copyright Michael Park, 2017
+  Copyright Agustin Berge, 2017
+
+  Distributed under the Boost Software License, Version 1.0.
+  (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
+*/
+
+#include <slb/utility.hpp>
+
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+TEST_CASE("integer_sequence", "[intseq.intseq]") {
+  /* using value_type = T; */ {
+    CHECK(std::is_same<slb::integer_sequence<int, -1, 1>::value_type,
+                       int>::value);
+  }
+
+  /* static constexpr std::size_t size() noexcept; */ {
+    using is = slb::integer_sequence<int, 0, 1, 3, 2>;
+    CHECK(std::is_same<decltype(is::size()), std::size_t>::value);
+    CHECK(noexcept(is::size()));
+    CHECK(is::size() == 4);
+    constexpr std::size_t size = is::size();
+    (void)size;
+  }
+}
```

Comment:
Needs checks for compatibility with `std::integer_sequence` when available. Since they will actually be the same type, it could be sufficient to just do a same type check for `std`/`slb`.